lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2054 @@
+{
```

Comment:
<p>Proč je tam to </p><pre class="ql-syntax" spellcheck="false">if tweet.text not in tweets_list
</pre><p>? Tweety můžu dostat víckrát?</p><p> </p><p>tweets_list by to chtělo vytvořit uvnitř funkce</p><p> </p><p>Zase bych vyhodit to <code>except</code>  - takhle to udělá víc škody než užitku :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='98'/>

Reply:
<p>jo, to bylo kdyby to běželo denně např., ale možná z toho bude úloha</p>